docs/tutorials/faq.md

```diff
@@ -123,9 +123,9 @@ configuration][buffering] and the docs for the
 [`.streaming.bufferingGoal`][StreamingConfiguration] configuration.
 
 Another factor is the segment size.  It may take up to 2 segments before Shaka
-Player has enough information to form a bandwidth estimate and make a decision.
+The Player has enough information to form a bandwidth estimate and make a decision.
```

Comment:
This isn't the beginning of a sentence. This would result in "Shaka The Player".

Reply:
Oh! I misread the sentence. Will fix.